index.bs

```diff
@@ -7580,6 +7580,61 @@ To <dfn abstract-op>Create a new supplemental public key record</dfn>, perform t
         [=set/append=] this [=supplemental public key record=] to |credentialRecord|.[$credential record/supplementalPubKeys$].
 
 
+### User verification caching extension (<dfn>userVerificationCaching</dfn>) ### {#sctn-user-verification-caching-extension}
+
+In some cases it is good enough for the [=[RP]=] to know whether the user was verified by the authenticator "recently".
+
+This extension allows the [=[RP]=] to specify such [=user verification=] caching time, i.e. the time for which the [=user verification=] status can be "cached" by the [=authenticator=].
+
+For example: Do not ask the user for a fresh [=user verification=] for sign-in if the user was verified by this authenticator within the past 300 seconds.
+
+: Extension identifier
+:: `userVerificationCaching`
+
+: Operation applicability
+:: [=authentication extension|authentication=]
+
+: Client extension input
+:: The maxTimeSinceLastUV denotes the maximum acceptable number of milliseconds elapsed since the last time the user was successfully verified.
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+      unsigned long long maxTimeSinceLastUV;
```

Comment:
This member name needs to be the same as the extension identifier.
```suggestion
      unsigned long long userVerificationCaching;
```

Reply:
Yes, `unsigned long long` is the [WebIDL type for a 64-bit integer](https://webidl.spec.whatwg.org/#idl-unsigned-long-long).

...oh, I was about to say we use the same type for [`timeout`](https://w3c.github.io/webauthn/#dom-publickeycredentialcreationoptions-timeout), but it turns out it's just `unsigned long` (32-bit). So yeah, let's make it just `unsigned long` here too for consistency. 2<sup>64</sup> milliseconds is almost 50 days, I can't imagine anyone needing more range than that.